benchmark/net/tcp-raw-s2c.js

```diff
@@ -110,12 +110,12 @@ function main({ dur, len, type }) {
     connectReq.oncomplete = function() {
       var bytes = 0;
       clientHandle.onread = function(buffer) {
-        // we're not expecting to ever get an EOF from the client.
+        // We're not expecting to ever get an EOF from the client.
         // just lots of data forever.
         if (!buffer)
           fail('read');
 
-        // don't slice the buffer.  the point of this is to isolate, not
+        // Don't slice the buffer.  the point of this is to isolate, not
```

Comment:
Same: I realize you probably did this with --fix, but while we're in here, is it worth it to capitlaize The and get rid of the two-spaces separation?